File: moxy/src/main/java/com/omegar/mvp/Moxy.java
package com.omegar.mvp;

import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.TYPE;

/**
 * Created by Anton Knyazev on 25.11.2020.
 */
@Target(value = TYPE)
public @interface Moxy {

    String reflectorPackage();

}

File: moxy/src/main/java/com/omegar/mvp/Moxy.kt
package com.omegar.mvp;

import static java.lang.annotation.ElementType.TYPE;

import java.lang.annotation.Target;

/**
 * Created by Anton Knyazev on 25.11.2020.
 */
@Target(value = TYPE)
public @interface Moxy {

    String reflectorPackage();

}
